tempel.el

```diff
@@ -315,7 +316,7 @@ Return the added field."
       (overlay-put ov 'face 'tempel-default)
       (overlay-put ov 'tempel--default
                    (if (string-suffix-p ": " init) 'end 'start)))
-    (tempel--synchronize-fields st ov)
+    (tempel--synchronize-fields (cons (cons nil (car st)) (cdr st)) ov)
```

Comment:
This also looks a bit hackish. Would it be possible to create the range overlay first and then move it repeatedly such that it always wraps the entire template?

Reply:
done